src/common/dataHooks/useStats.js

```diff
@@ -0,0 +1,73 @@
+import { useQuery } from '@tanstack/react-query'
+
+// FPCC
+import { STATS } from 'common/constants'
+import api from 'services/api'
+import { useParams } from 'react-router-dom'
+
+export function useStats() {
+  const { sitename } = useParams()
+  const response = useQuery([STATS, sitename], () =>
+    api.stats.get({ sitename }),
+  )
+  const types = [
+    'words',
+    'phrases',
+    'songs',
+    'stories',
+    'images',
+    'audio',
+    'video',
+  ]
+  const temporal = response?.data?.temporal
+  const aggregate = response?.data?.aggregate
+
+  const getPeriodTotalForType = (period, modelType) =>
+    temporal?.[modelType]?.[period]?.created
+
+  const getPeriodTotal = (period) => {
+    let total = 0
+    types.forEach((type) => {
+      total += getPeriodTotalForType(period, type)
+    })
+    return total
+  }
+
+  const getPeriodTotalsForAllTypes = (period) => {
+    const totals = {}
+    types.forEach((type) => {
+      totals[type] = getPeriodTotalForType(period, type)
+    })
+    return totals
+  }
+
+  const formattedStats = () => {
+    const weekTotal = getPeriodTotal('last7Days')
+    const monthTotal = getPeriodTotal('lastMonth')
+
+    if (weekTotal > 4) {
```

Comment:
The stats widget should work the same as when the data was being provided by Nuxeo, however I have been unable tto trigger the widget into displaying "new this week", "new this month" and "on this site" are working correctly though

Reply:
The API returns "lastWeek" not "last7Days"... does this time period string need to match?

Reply:
That was it! The display is now fixed, thank you!